### DISCUSSION TITLE
我的世界基岩1.21服务器有人来玩吗？

### DISCUSSION THREAD

**Discussion by QCG1081404q** [share121/inter-knot#2946] [其他]
我的世界基岩1.21服务器有人来玩吗？
![Screenshot_20240812_020457_com mojang minecraftpe](https://github.com/user-attachments/assets/193408d9-fcee-423f-8137-528c966b8551)
![Screenshot_20240812_020818_com mojang minecraftpe](https://github.com/user-attachments/assets/375d690b-2544-4c8e-836d-9a2530c2fdba)
![Screenshot_20240816_000410_com mojang minecraftpe](https://github.com/user-attachments/assets/9917b5b0-7e52-4f3e-9bee-91362613495d)
![mmqrcode1724003540890](https://github.com/user-attachments/assets/d3b0471c-f232-42bf-8eb6-80ff13d9d9c9)

![emotion_download_1723913416831](https://github.com/user-attachments/assets/6b8b90ae-ac97-43ae-aa75-c2d9b2b2886f)

**Comment by OttoApocalypse01** [comment]
装了模组吗<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3996afc68a13bfc5b5718af31e610f43_6807527097924316704.png" alt="罗刹 送花" width="75" >

**Comment by QCG1081404q** [comment]
大部分是纯生存，因为模组的话最新的BDS并不支持，我找了半天也没有找到解决办法，所以现在服务器是纯生存。。
      呃，来玩吗？

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by QCG1081404q** [comment]
大部分是纯生存，因为模组的话最新的BDS并不支持，我找了半天也没有找到解决办法，所以现在服务器是纯生存。。
      呃，来玩吗？


________________________________

**Comment by lllliufeng** [comment]
可恶啊，不想启动基岩版（

**Comment by cpu644** [comment]
你这个beta一般基岩也进不了吧。

**Comment by ybbgh** [comment]
基岩版算了吧<img src="https://bbs-static.miyoushe.com/static/2023/05/23/a2fa805a0a782cdb8ab44ddaf8d5343d_962898009752203374.png" alt="阿姬倒地了04" width="75" >

**Comment by QCG1081404q** [comment]
为了适合这个版本，我倒是会分发安装包（APK）
________________________________
